src/components/DeckMechanisms/handleCardEffect.js

```diff
@@ -301,9 +302,13 @@ const handleCardEffect = (state, card, mode) => {
 
     // Harvesters of Souls
     case 'N38': {
-      const copiedCard = getHarvestersOfSoulsCopiedCard(state, card.level)
-      if (copiedCard) {
-        state.deck.push(copiedCard)
+      const copiedCards = Array.from(
+        { length: HARVESTERS_OF_SOULS_RNG.POTENTIAL_CARDS[state.RNG] },
+        () => getHarvestersOfSoulsCopiedCard(state, card.level)
+      ).filter(Boolean)
+      if (copiedCards.length) {
+        harvestersActions.setCards(copiedCards)
```

Comment:
We’ll need to guarantee a single faction + neutral amongst the provided cards. Unless playing against another HoS, there is no way to have cards from different factions.